src/wurf/context_msg_resolver.py

```diff
@@ -26,9 +24,10 @@ def resolve(self):
 
         :return: The path as a string.
         """
+        
 
         self.ctx.start_msg('{} {}'.format(
-            self.method, self.dependency.name))
+            self.dependency.resolver_method, self.dependency.name))
```

Comment:
Yes, this is beyond the scope of this branch. I will experiment with this in another branch.